packages/story-editor/src/app/highlights/provider.js

```diff
@@ -58,10 +58,11 @@ function HighlightsProvider({ children }) {
       }
 
       if (highlight) {
-        const { tab, ...highlightState } = STATES[highlight];
+        const { tab, subTab, ...highlightState } = STATES[highlight];
```

Comment:
Ugh, this is getting ugly, but I guess there's no good way around it. "subTab" doesn't sound good though, IMO. But I have no better ideas. 🤷

Reply:
I guess we can do something like:
```
tab: `${INSERT}/${PAGE_TEMPLATES.id}`
```
without `subTab`, would that be any better?

Reply:
what about `section` instead of `subTab`?

Reply:
`section` is ok, it might be a little confusing that `section` leads to a pane with multiple `section` elements (there is no 1:1 connection between them), but I guess that's fine.

![image](https://user-images.githubusercontent.com/21036224/156552940-a1ca039d-3e7f-4404-981e-129ff5593004.png)

Reply:
Done in https://github.com/GoogleForCreators/web-stories-wp/pull/10760/commits/872b439e308ef93caad29d4a3d70b90a64ab21cb